lib/lita/adapters/slack/user_creator.rb

```diff
@@ -11,6 +11,7 @@ def create_user(slack_user, robot, robot_id)
             )
 
             update_robot(robot, slack_user) if slack_user.id == robot_id
+            robot.trigger(:slack_user_created, user_id: slack_user.id, raw_data: slack_user.raw_data)
```

Comment:
I'd considered that too. To be honest, the only thing I want is the user id, as I can re-find the user with that. I added the `raw_data` to help the other folks asking for such things. In any case, I do wonder if we should at least put the raw data under a known key to make it easier to enumerate. e.g.,

```ruby
robot.trigger(:slack_user_created, slack_data: slack_user.raw_data)
```